src/transform/schema.ts

```diff
@@ -95,14 +108,16 @@ export function transformSchemaObj(node: any, options: TransformSchemaObjOptions
       }
       case "object": {
         const isAnyOfOrOneOfOrAllOf = "anyOf" in node || "oneOf" in node || "allOf" in node;
-
+        const missingRequired = addRequiredProps(node.properties || {}, node.required || []);
         // if empty object, then return generic map type
         if (
           !isAnyOfOrOneOfOrAllOf &&
           (!node.properties || !Object.keys(node.properties).length) &&
           !node.additionalProperties
         ) {
-          output += `{ ${readonly}[key: string]: any }`;
+          const emptyObj = `{ ${readonly}[key: string]: any }`;
```

Comment:
Same here: 
```suggestion
          const emptyObj = `{ ${readonly}[key: string]: unknown }`;
```

Reply:
Thinking about this, this might be a breaking change for a lot of folks, who use empty projects and rely on the any.  In my project, this wouldn't be a problem. 

I will include this change for now. Let me know your thoughts

Reply:
I’m really sorry but I think you do raise a good point and you’ve changed my mind on this—this actually would cause disruption for people. And #554 didn’t really get a lot of support from people, so I’m starting to second-guess it. This PR shouldn’t be blocked by another that may or may not merge.

It seems there is another merge conflict (apologies); if you’re able to update that would you mind changing this back to `any` again? Again I’m sorry and this is 100% on me. If I don’t hear from you soon, I can update this PR myself and still give you credit for it. Consider this PR ready-to-go!